packages/core/app/helpers/ember-array.js

```diff
@@ -1 +1,9 @@
-export { default, emberArray } from 'ember-composable-helpers/helpers/array';
+import { helper } from '@ember/component/helper';
```

Comment:
Do we still need this? GH shows only one usage of `ember-array`. Can we just use: https://api.emberjs.com/ember/3.17/classes/Ember.Templates.helpers/methods/array?anchor=array

Reply:
Yes we still use it at least in the `report-visualization`
https://github.com/yahoo/navi/blob/cec92ce52d1937f10a63633c48cb096b703a37c4/packages/reports/addon/templates/components/report-visualization.hbs#L5
So this would require refactoring our `navi-visualizations/`

We can make an issue to refactor them

Reply:
https://github.com/yahoo/navi/issues/726